common/src/main/kotlin/org/jetbrains/research/deepbugs/common/DeepBugsConfig.kt

```diff
@@ -8,8 +8,16 @@ abstract class DeepBugsConfig(val configId: String, private val default: State)
     data class State(
         @Property val binOperatorThreshold: Float = 0.0f,
         @Property val binOperandThreshold: Float = 0.0f,
-        @Property val swappedArgsThreshold: Float = 0.0f
-    )
+        @Property val swappedArgsThreshold: Float = 0.0f,
+        @Property val userDisabledChecks: HashSet<String> = HashSet()
+    ) {
+        fun copy(): State = State (
```

Comment:
I would name it `clone`, since otherwise it overwrites `copy` of data class, but not fully